spec/features/provider_login_spec.rb

```diff
@@ -53,11 +53,9 @@
         body: build(:api_result, eligible: "eligible").to_json,
         headers: { "Content-Type" => "application/json" },
       )
-      first = instance_double(CfeResult, ineligible_gross_income?: true,
-                                         gross_income_excess: 100,
+      first = instance_double(CfeResult, gross_income_excess: 100,
                                          gross_income_result: "ineligible")
```

Comment:
would it be worth renaming first and second to ineligible_result and eligible_result for clarity?